packages/lesswrong/components/votes/EAEmojisVoteOnComment.tsx

```diff
@@ -27,299 +17,8 @@ const styles = (theme: ThemeType): JssStyles => ({
       },
     },
   },
-  button: {
```

Comment:
All the code deleted in this file just moved into `EAReactsSection` for reuse elsewhere